README.md

```diff
@@ -10,3 +10,31 @@ This is an express endpoint which calls ElasticSearch and returns the results.
 # Search page
 This is a small react app using redux.
 
+# Development
+
+### setup blockbuilder dev environment
+
+Follow the [full dev setup guide](https://github.com/enjalot/blockbuilder/wiki/Development#development) at the **blockbuilder** code-editor project
+
+Once you've done that, verify that it worked by opening a web browser and visiting local blockbuilder at [http://[::]:8889](http://[::]:8889)
+
+### setup search index backend
+
+Follow the steps in the [Setup Elasticsearch & Index some Gists](https://github.com/enjalot/blockbuilder-search-index#setup-elasticsearch--index-some-gists) section of the **blockbuilder-search-index** project
+
+These steps guide us through setting up, installing, and running a local Elasticsearch instance that we can populate with blocks.  The blocks data is retrieved from Github and stored as static json files, before being loaded into our Elasticsearch index.  Once our search index is up, running, and populated, we're reading to query it with a web ui. 
+
+### connect local blockbuilder with local blockbuilder-search
+
+```
+// npm link blockbuilder-search
```

Comment:
We need 2 steps here:
cd blockbuilder-search
npm link
cd ../blockbuilder
npm link blockbuilder-search

I don't usually use // when documenting terminal commands. should be able to do ```bash

Reply:
that's right `# bash comment`  

got lazy there 😉